backend/config/runtime.exs

```diff
@@ -81,8 +81,8 @@ case System.fetch_env!("FILE_STORAGE_ADAPTER") do
   "s3" ->
     IO.puts("Setup S3 file storage")
     s3_host = System.get_env("S3_HOST") || ""
-    s3_key_id = System.get_env("S3_KEY_ID") || ""
-    s3_key_secret = System.get_env("S3_KEY_SECRET") || ""
+    s3_key_id = System.get_env("S3_KEY_ID") || System.get_env("S3_ACCESSS_KEY") || ""
+    s3_key_secret = System.get_env("S3_KEY_SECRET") || System.get_env("S3_SECRET_KEY") || ""
```

Comment:
Heroku addons defaut suffixes. Did not find how to define our suffixes

Reply:
Would be great to have a comment in the code to explain this as well ;)